controllers/operator/construct/database_construction_test.go

```diff
@@ -45,6 +45,9 @@ func Test_buildDatabaseInitContainer(t *testing.T) {
 		SecurityContext: &corev1.SecurityContext{
 			ReadOnlyRootFilesystem:   ptr.To(true),
 			AllowPrivilegeEscalation: ptr.To(false),
+			Capabilities: &corev1.Capabilities{
```

Comment:
is there any potential that adding this default will break *any* customer's workload and will require some manual intervention? Just thinking about our semver guarantees.

Reply:
This is our defaults. Only problem I see with this is some customers now requiring explicitly setting capabilities.

Reply:
Still, do we force customers to do any manual fix when upgrading? If yes, we need to bump major.

Reply:
I don't understand what will we break here. We are changing our default SecurityContext for operator and other pods created. If customer wants to have dedicated SecurityContext or PodSecurityContext they need to set `MANAGED_SECURITY_CONTEXT` env var and our defaults will be completely overwritten. If they don't set `MANAGED_SECURITY_CONTEXT` every change they make to SecurityContext manually will be overwritten by our defaults.